CMakeLists.txt

```diff
@@ -7,14 +7,17 @@ project (camp
 # Default to C++11 if not set so GTest/GMock can build
 if (NOT BLT_CXX_STD)
     set(BLT_CXX_STD "c++14" CACHE STRING "")
+else()
+    if (${BLT_CXX_STD} STREQUAL "c++11")
```

Comment:
BLT_CXX_STD also allows c++89, so you might want to check for that too.

Reply:
Good point, I'll do that.   I think we have the same check in RAJA, so we may want to propagate it there as well.

Reply:
Yup, I have this check in the Umpire C++14 PR: https://github.com/LLNL/Umpire/pull/633/